spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/RetryingMessageListenerAdapter.java

```diff
@@ -37,7 +40,7 @@
  */
 public class RetryingMessageListenerAdapter<K, V>
```

Comment:
Looks like we are missing `RetryingBatchMessageListenerAdapter` and there is no way to retry the whole batch.
Or is that by design do not encourage end-users to retry on the whole batch ?
Thanks